src/main/java/Duke.java

```diff
@@ -1,10 +1,83 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private static ArrayList<Task> taskList = new ArrayList<>();
```

Comment:
According to the Java coding standard guide, under the Naming section, they mentioned that naming of constant variables must be all uppercase using underscore to separate words. Hence should this be ` private static ArratList<Task> TASK_LIST` instead of `private static ArrayList<Task> taskList`?

Reply:
Personally, I think naming it as taskList is alright here. I think in the textbook when they are referring to constant variables, it's more like fixed values e.g. int MATH_PI = 3.14 . In this context, I would consider taskList more of a variable rather than a constant instead. :)

Reply:
Yea, thank you both as taskList is meant to be a variable here because it will get updated